index.html

```diff
@@ -34,271 +35,308 @@
+                                    <li class="card-tag-crystal">html</li>
+                                    <li class="card-tag-crystal">bootstrap</li>
+                                    <li class="card-tag-crystal">ruby</li>
+                                </ul>
+                                <div class="mpfooter">
+                                    <a onclick="openModal()">See Project</a>
+                                </div>
                             </div>
                         </div>
-                    </div>
-        
-                    <!--Normal cards-->
-                    <div class="card">
-                        <h2 class="card-title">Profesional Art Printing Data</h2>
-                        <div class="card-body">
-                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                            <ul class="tags">
-                                <li class="card-tag">html</li>
-                                <li class="card-tag">bootstrap</li>
-                                <li class="card-tag">ruby</li>
-                            </ul>
-                        </div>
-                        <div class="card-footer">
-                            <a href="https://wikitelmex.github.io/Portfolio/">See Project</a>
-                        </div>
-                    </div>
             
-                    <div class="card">
-                        <h2 class="card-title">Profesional Art Printing Data</h2>
-                        <div class="card-body">
-                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                            <ul class="tags">
-                                <li class="card-tag">html</li>
-                                <li class="card-tag">bootstrap</li>
-                                <li class="card-tag">ruby</li>
-                            </ul>
+                        <!--Normal cards-->
+                        <div class="card">
+                            <h2 class="card-title">Profesional Art Printing Data</h2>
+                            <div class="card-body">
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                <ul class="tags">
+                                    <li class="card-tag">html</li>
+                                    <li class="card-tag">bootstrap</li>
+                                    <li class="card-tag">ruby</li>
+                                </ul>
+                            </div>
+                            <div class="card-footer">
+                                <a href="https://wikitelmex.github.io/Portfolio/">See Project</a>
+                            </div>
```

Comment:
This part of your code should be generated dynamically using JavaScript, you should create a constant like this:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
Note: With all of your projects, and then you should display these projects dynamically using JavaScript, you can use the "forEach()" method and "innerHTML" to do it 😉 

**Feel free to check the example of students projects:**
[Example 1](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)
[Example 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96)

- [ ] Your web app should be dynamically displaying the projects.